Common/SonarQube.Plugins.Common.csproj

```diff
@@ -72,6 +72,9 @@
     <Compile Include="ConsoleLogger.cs" />
     <Compile Include="Interfaces\ILogger.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="..\AssemblyInfo.Shared.cs">
+      <Link>Properties\AssemblyInfo.Shared.cs</Link>
+    </Compile>
```

Comment:
All of the projects have been updated to link in the shared file (except the SonarLint project, which will be deleted)